java/gast_to_code/gast_to_code_java.py

```diff
@@ -27,7 +27,20 @@ def handle_log_statement(gast):
         return "System.out.println"
 
     def handle_var_assign(gast):
-        pass
+        var_id = router.gast_to_code(gast["varId"], "js")
+        var_value = router.gast_to_code(gast["varValue"], "js")
+        var_type = gast["varValue"]["type"]
+        
+        if var_type == "num":
+            kind = "int"
+        elif var_type == "str":
+            kind = "String"
+        elif var_type == "bool":
+            kind = "boolean"
+        else:
```

Comment:
Should this be an unsupported error instead? At least for the type part.

Reply:
I think this is for when the kind is not a primitive i.e. MyFooClass. Can we store/use that exact type instead? i.e. 

java -> java would be
MyFooClass foo = new FooClass(); -> MyFooClass foo = new FooClass(); 
instead of customType foo = new FooClass();